recipe/meta.yaml

```diff
@@ -20,16 +20,17 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.4
+    - python
 
 test:
   imports:
     - imagesize
   requires:
     - pip
-    - python <3.10
   commands:
     - pip check
+  downstreams:
+    - sphinx  # [not py==310]
```

Comment:
we shouldn't use downstream feature, please comment

Reply:
Sure, I agree. Downstream is a great feature for a non-production building stage. I'll remove it